### DISCUSSION TITLE
how to build the the rxfnsb library

### DISCUSSION THREAD

**Discussion by Peter-Jacob** [adesutherland/CREXX#341] [Q&A]
I am working on the symbol bif. To run it it must be added to rxfnsb (I assume). How can I do that, please enlighten me!

**Comment by rvjansen** [comment]
It is all in the CMakeLists.txt file, if you treat it like the others it will be added to the library.

**Comment by Peter-Jacob** [comment]
I added symbol to it, ran a build, now it works
